OpenBudgeteer.Core/ViewModels/ItemViewModels/TransactionViewModelItem.cs

```diff
@@ -267,6 +267,22 @@ public static async Task<TransactionViewModelItem> CreateFromBucketMovementAsync
         return await Task.Run(() => new TransactionViewModelItem(bucketMovement));
     }
 
+    
+    /// <summary>
+    /// Adds a bucket item.
+    /// </summary>
+    /// <param name="amount">Amount that will be assigned to the Bucket</param>
+    /// <param name="newBucketItem">Amount that will be assigned to the Bucket</param>
```

Comment:
Can you please check documentation here. It has the same explanation as `amount`